src/web/index.mdx

```diff
@@ -25,10 +25,10 @@ Copy the tracking code from your project settings and include it on your web. Th
 
 Check the `GDPR compliant by design` section and adjust the settings if needed. If you plan to identify visitors from EU you have to sign our DPA, read more on that in the GDPR section.
 
-
 ## Further Steps
 
 <TextBlock>{`
+* Check our <Link target="_blank" to="https://help.smartlook.com">Knowledge base</Link> for more general information, integrations and FAQ.
```

Comment:
I think we can remove **TextBlock** here, also change **Link** with html **a** tag as It's not a internal app link

Reply:
done

Reply:
cool